doc/source/actors.rst

```diff
@@ -10,32 +10,63 @@ that specific worker and can access and mutate the state of that worker.
 Creating an actor
 -----------------
 
-You can convert a standard Python class into a Ray actor class as follows:
+.. tabs::
+  .. group-tab:: Python
 
-.. code-block:: python
+    You can convert a standard Python class into a Ray actor class as follows:
 
-  @ray.remote
-  class Counter(object):
-      def __init__(self):
-          self.value = 0
+    .. code-block:: python
 
-      def increment(self):
-          self.value += 1
-          return self.value
+      @ray.remote
+      class Counter(object):
+          def __init__(self):
+              self.value = 0
 
-Note that the above is equivalent to the following:
+          def increment(self):
+              self.value += 1
+              return self.value
 
-.. code-block:: python
+    Note that the above is equivalent to the following:
 
-  class Counter(object):
-      def __init__(self):
-          self.value = 0
+    .. code-block:: python
 
-      def increment(self):
-          self.value += 1
-          return self.value
+      class Counter(object):
+          def __init__(self):
+              self.value = 0
 
-  Counter = ray.remote(Counter)
+          def increment(self):
+              self.value += 1
+              return self.value
+
+      Counter = ray.remote(Counter)
+
+  .. group-tab:: Java
+
+    Unlike Python, you don't need to modify your Java class to make it a Ray actor class. You can create an actor with a constructor of the Java class or a factory method.
```

Comment:
I prefer removing "Unlike Python, you don't need to modify your Java class to make it a Ray actor class.". Because new users might only be interested in Java. This would be confusing for them.
Just "You can convert a standard Java class into a Ray actor class as follows:" is fine.